### PR TITLE
feature/HW-64673 manage transfer method ux update

### PR DESCRIPTION
**Pull request by peter-joseph**

### CODE REVIEW

File: transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodActivity.java [hunk 1]
```diff
@@ -68,7 +70,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
         getSupportActionBar().setDisplayShowHomeEnabled(true);
-        getSupportActionBar().setTitle(R.string.title_activity_list_transfer_method);
+        getSupportActionBar().setTitle(R.string.mobileTransferMethodsHeader);
```
Comment (jsaini-hw) [inline transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodActivity.java:73]: I see 2 changes in this PR for header `title_activity_list_transfer_method` and `mobileTransferMethodsHeader`. Remove the one that's not used

File: commonui/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -31,4 +30,9 @@
     <string name="not_translated_in_braces">(not translated)</string>
 
     <string name="authentication_error_header">Authentication Error</string>
+
+    <string name="mobileTransferMethodsHeader">Transfer methods</string>
```
Comment (jsaini-hw) [inline commonui/src/main/res/values/strings.xml:34]: Ideally this should not be commonUI module but inside Transfer Method module

Comment (peter-joseph) [reply commonui/src/main/res/values/strings.xml:34]: yeah but it's used in transfers ui and transfer method account ui modules

File: transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodActivity.java (hunk 1 above)
Comment (peter-joseph) [reply transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodActivity.java:73]: `title_activity_list_transfer_method` this is just for manifest

Comment (peter-joseph) [reply transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodActivity.java:73]: ok looks like duplicating the keys on every other module just works, removing the other keys and just use one single key `mobileTransferMethodsHeader`

**Pull request merged**